examples/agent/step_wise_openai.ts

```diff
@@ -69,11 +70,24 @@ async function main() {
   for await (const stepOutput of task) {
     console.log(`Runnning step ${count++}`);
     console.log(`======== OUTPUT ==========`);
-    console.log(stepOutput.output.message.content);
+    const output = stepOutput.output;
+    if (output instanceof ReadableStream) {
+      for await (const chunk of output) {
+        process.stdout.write(chunk.delta);
+      }
+    } else {
+      console.log(output);
+    }
```

Comment:
This seems ugly to use, but since this not for user side API, I think it's okay for now